net-analyzer/ossec-hids/ossec-hids-3.1.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user
+
+DESCRIPTION="Open Source Host-based Intrusion Detection System"
+HOMEPAGE="https://www.ossec.net/"
+SRC_URI="https://github.com/ossec/ossec-hids/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2"
```

Comment:
Please keep variables ordered and grouped as in template; it makes it easier to read ebuilds when we know where to expect vars.

Also, have you verified whether it's not GPL-2+? [or newer]

Reply:
The variables are already ordered as in skel.ebuild, but I'll adjust the grouping. Regarding the license, I looked at [this document](https://github.com/ossec/ossec-hids/blob/master/LICENSE).

Reply:
In my previous comment I linked to the `LICENSE` file that's contained in the tarball. Individual files repeat that the software is copyrighted by Trend Micro and that GPL version 2 applies. I see no mention of other GPL versions.

Reply:
Ok, thanks for verifying.